.unreleased/LLT-5663

```diff
@@ -0,0 +1 @@
+Fix TTC for wg peers
```

Comment:
Expant TCC so it would be clear in the future

Reply:
Fixed